### DISCUSSION TITLE
collection/list-war-2

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#148] [Announcements]
# collection/list-war-2

Java程序员进阶之路，小白的零基础Java教程，Java中ArrayList和LinkedList的区别

https://tobebetterjavaer.com/collection/list-war-2.html

**Comment by 2258324319** [comment]
"不能像某些东西一样伸缩自如。"???

**Reply by itwanger** [reply]
奇怪的发现

**Reply by huamus** [reply]
啊哈哈哈
